src/components/messenger/list/group-management/container.tsx

```diff
@@ -79,7 +79,8 @@ export class Container extends React.Component<Properties> {
         profileImage: currentUser?.profileSummary.profileImage,
         matrixId: currentUser?.matrixId,
         isOnline: currentUser?.isOnline,
-        primaryZID: getUserHandle(currentUser?.primaryZID, currentUser?.wallets),
+        primaryZID: currentUser?.primaryZID,
+        primaryZIDDisplayHandle: getUserHandle(currentUser?.primaryZID, currentUser?.wallets),
```

Comment:
so just remove the `primaryZID` field from here, and use `displaySubHandle` instead of `primaryZIDDisplayHandle`?

Reply:
I mean remove `primaryZID` from `primaryZIDDisplayHandle`. The intent of the variable is to hide what it's built from so including that at the start of the property name defeats the purpose. But then my secondary thought was... perhaps `displayHandle` is wrong since we actually use the user's name as their display and this is more like a subtitle.

Reply:
got it.